src/contractTest/resources/application.properties

```diff
@@ -77,6 +77,8 @@ service.pdf-service.health.uri=${PDF_SERVICE_HEALTH_URL:https://docmosis.aat.pla
 service.pdf-service.uri=${PDF_SERVICE_BASE_URL:https://docmosis.aat.platform.hmcts.net/rs/render}
 spring.application.name=sscs evidence share
 spring.main.allow-bean-definition-overriding=true
+springdoc.packages-to-scan=*.controllers
+springdoc.paths-to-match=*
```

Comment:
Unsure what to put here. Attempting to follow [these instructions](https://springdoc.org/migrating-from-springfox.html#:~:text=If%20you%20have%20only%20one%20Docket%E2%80%89%E2%80%94%E2%80%89remove%20it%20and%20instead%20add%20properties%20to%20your%20application.properties)

Reply:
If you only want the controllers package, you could change the "*" to "uk.gov.hmcts.reform.sscs.controllers", otherwise there is a tiny chance where we'd need to add a different controllers package in future that would also be picked up by this scanning. Makes it future proof.